Makefile

```diff
@@ -154,12 +152,6 @@ azure-ipam-version: ## prints the azure-ipam version
 cni-version: ## prints the cni version
 	@echo $(CNI_VERSION)
 
-cni-dropgz-version: ## prints the cni-dropgz version
```

Comment:
I see in the following files, we reference this `make cni-dropgz-version` target, to make a dropgz version to run integration tests with
We would probably just use `make cni-version` instead right?
- `.pipelines/singletenancy/aks-swift/e2e-step-template.yaml`
- `.pipelines/singletenancy/azure-cni-overlay/azure-cni-overlay-e2e-step-template.yaml`
- `.pipelines/singletenancy/cilium/cilium-e2e-step-template.yaml`
- `.pipelines/singletenancy/cilium-overlay/cilium-overlay-e2e-step-template.yaml`
- `.pipelines/singletenancy/dualstack-overlay/dualstackoverlay-e2e-step-template.yaml`

Reply:
And then just a reminder we'll also have to change this variable name
https://github.com/Azure/azure-container-networking/blob/beeb66aaddd1dee9e7bbaae5df93297a0447aa3b/test/internal/kubernetes/utils_create.go#L51C45-L51C48

Reply:
Well, we might not need to have to update the variable name I linked, but we will need to adjust the pipeline `.yaml` files I listed there, to use `make cni-version`